install-dev/data/db_structure.sql

```diff
@@ -376,6 +376,7 @@ CREATE TABLE `PREFIX_cms_lang` (
   `id_lang` int(10) unsigned NOT NULL,
   `id_shop` int(10) unsigned NOT NULL DEFAULT '1',
   `meta_title` varchar(128) NOT NULL,
+  `head_seo_title` varchar(128) DEFAULT NULL,
```

Comment:
Is there a reason to have only 128 characters?

Reply:
Yes, the optimal length for a title tag is 50–60 characters. Therefor 128 should be more than enough :-)

Reply:
Proof: https://moz.com/learn/seo/title-tag

Reply:
Optimal doesn't mean it's mandatory.
More don't see anything to check if we have a string higher than 128 characters :/
What happened if we write 500 characters in field?

Reply:
I see your point, a check could be good to have in order to make it more userfriendly.

Google don't show more than 600 px. Therefor you will not get any value of more than 128 characters.

Reply:
Google isn't the only one who provide a search engine and the title can be used by humans :trollface: 
Like meta_description (optimal length of 110/120) it could be better to use a VARCHAR(255)
wdyt @eternoendless ?

Reply:
:trollface:

Reply:
No all (for update reasons, not necessary for the moment), but at least what we added. @eternoendless said to @matks to set VARCHAR to 255 when it's possible.

Reply:
ok! done